src/linked_list.py

```diff
@@ -0,0 +1,79 @@
+"""The Linked List Module Creates a LinkedList."""
+
+
+class Node(object):
+    """Defining Node Class."""
+
+    def __init__(self, val, next=None):
+        """Instantiate Node Class."""
+        self.val = val
+        self.next = next
+
+
+class LinkedList(object):
+    """Defining class of LinkedList."""
+
+    def __init__(self, iterable=None):
+        """Instantiate LinkedList class."""
+        self.head = None
+        self.size_of_list = 0
+        if iterable and hasattr(iterable, "__iter__"):
+            for value in iterable:
+                self.push(value)
+        elif iterable:
+            self.push(iterable)
```

Comment:
So if it's an iterable, you push every element and if it's not an iterable you push it anyway? That's not really in line with the spec, which specifically says that it should take an optional iterable on instantiation.